nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_13TEV/filter.py

```diff
@@ -0,0 +1,96 @@
+"""
+filter.py module for ATLAS_WPWM_13TEV dataset
+When running `python filter.py` the relevant uncertainties , data and kinematics yaml
+file will be created in the `nnpdf_data/commondata/ATLAS_WPWM_13TEV` directory.
+"""
+
+import yaml
+from filter_utils import get_data_values, get_systematics
+from nnpdf_data.filter_utils.utils import prettify_float
+
+yaml.add_representer(float, prettify_float)
+
+MZ2 = 91.1876**2
```

Comment:
In this case I literally just took the value that was being used in the legacy version.

But this is a good point. I agree with you that it would be nice to collect this values / constants in a file (eg filter_utils) so as to use them in a consistent way..

Reply:
Sorry I thought I had already replied to this.
I think its a good idea in principle.
Here I really just took the value that was already used in the legacy version for the kinematics